pkg/builder/rest/rest.go

```diff
@@ -41,7 +41,7 @@ type StaticHandlerProvider struct {
 
 // Get returns itself as the handler
 func (p StaticHandlerProvider) Get(s *runtime.Scheme, g generic.RESTOptionsGetter) (rest.Storage, error) {
-	return p, nil
+	return p.Storage, nil
```

Comment:
and this fixes custom storage that `StaticHandlerProvider` wont inherit `rest.Getter/Creator/Updater` interface from its composed rest.Storage interface